config/default.yml

```diff
@@ -159,6 +159,7 @@ Rails/DynamicFindBy:
   VersionAdded: '0.44'
   Whitelist:
     - find_by_sql
+    - Gem::Specification
```

Comment:
I wonder if methods & classes should be separated like:

```yaml
Whitelist:
  Methods:
    - find_by_sql
  Classes:
    - Gem::Specification
```

Reply:
Yeah, they will preferably be separated 👍 
OTOH, `Whitelist` will be deprecated in the future and RuboCop Rails is a major version, so suddenly breaking changes are not preferred.
It might be better to add the following two items in this PR.

```diff
+# The `Whitelist` has been deprecated, Please use `AllowedMethods` instead.
 Whitelist:
   - find_by_sql
+AllowedMethods:
+  - find_by_sql
+AllowedReceivers:
+  - Gem::Specification
```

The same applies to names of modules and methods to be added.

Reply:
Which means currently we should support both `WhiteList` & `Allowed..` ?

Reply:
If `Whitelist` is suddenly changed to `AllowedMethods`, users who has `Whitelist` in .rubocop.yml will encounter sudden breaking change.

Reply:
@koic Added support for `AllowedMethods` & `AllowedReceivers`.